src/util/time.h

```diff
@@ -6,24 +6,43 @@
 #ifndef BITCOIN_UTIL_TIME_H
 #define BITCOIN_UTIL_TIME_H
 
+#include <compat.h>
+
+#include <chrono>
 #include <stdint.h>
 #include <string>
-#include <chrono>
+
 #include <unordered_map>
+#include <sync.h>
```

Comment:
It should be possible to remove this include. It looks like it's not used here.

Reply:
It actually is used... by my MilliTimer class... there is a CCriticalSection there.

Reply:
You could probably get away with a forward declaration of CCriticalSection at the top and then just include the sync.h in time.cpp. Let me try that.

Reply:
Nope. Going to have to stay.